frontend/public/module/k8s/index.ts

```diff
@@ -117,8 +117,6 @@ export type K8sResourceKind = {
   apiVersion?: string;
   kind?: string;
   metadata?: ObjectMetadata;
-  // FIXME: This should not be in the generic `K8sResourceKind` definition.
-  pipelineTaskName?: string;
```

Comment:
Yikes :/ This was sloppy - I added this without thinking a while back, blindly fixing a typing issue I saw in test data. Pipelines shouldn't be referenced in the K8sResourceKind. Sorry @spadgett - it'll be removed by this PR.